lib/android/app/src/main/java/com/reactnativenavigation/viewcontrollers/toptabs/TopTabsController.java

```diff
@@ -22,12 +23,17 @@ public TopTabsController(Activity activity, String id, List<ViewController> tabs
     @NonNull
     @Override
     protected ViewGroup createView() {
-        return new TopTabsViewPager(getActivity(), tabs);
+        return new TopTabsLayout(getActivity(), tabs);
     }
 
     @NonNull
     @Override
     public Collection<ViewController> getChildControllers() {
         return tabs;
     }
+
+    @Override
+    protected boolean isViewShown() {
+        return super.isViewShown();
```

Comment:
is this correct?

Reply:
Not entirely,
I think `ParentViewController` should check if all child controllers are shown, wdyt?

Reply:
hmm.. what about sideMenu controller and other composable parents? not all child controllers are shown yet the parent is show..

Reply:
Will be addressed in #1083